dspace-server-webapp/src/test/java/org/dspace/app/rest/authorization/GenericAuthorizationFeatureIT.java

```diff
@@ -209,214 +209,214 @@ private void testAdminsHavePermissionsAllDso(String feature) throws Exception {
         String siteId = ContentServiceFactory.getInstance().getSiteService().findSite(context).getID().toString();
 
         // Verify the general admin has this feature on the site
-        getClient(adminToken).perform(get("/api/authz/authorizations/search/object?embed=feature&uri="
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/object?size=1000&embed=feature&uri="
```

Comment:
As said earlier by @LucaGiamminonni  here: https://github.com/DSpace/DSpace/pull/8797#discussion_r1197829921.
This fix is needed because some tests were failing inside this class, at the time he completed the PR.

The main reason is that every tests is checking for a specific feature by sending a standard request to the `authorization-feature` endpoint. However if not specified the rest endpoint will reply with only the first 20 entries, indeed if the entry that we are checking is in the second page of the pagination response (i.e. it's the 21st element), the test will wrongly fail.

Since @LucaGiamminonni fixed this test class during the first review phase, I think it would be better to leave this fix with a small refactoring to create a new method that will receive that large size as default for every request of that kind. In this way we can easily change the number of entities to load.